CHANGELOG.md

```diff
@@ -0,0 +1,9 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+[Unreleased]: https://github.com/MetaMask/metamask-module-template/
```

Comment:
This URL needs an update

Would be nice to add the lint changelog script as well!

Reply:
Oh, good point. I can add that to #17.